SimPEG/Utils/__init__.py

```diff
@@ -1,10 +1,18 @@
-from matutils import *
-from codeutils import *
-from meshutils import *
+from matutils import (mkvc, sdiag, sdInv, speye, kron3, spzeros, ddx, av,
```

Comment:
or do we prefer leaving these as wildcard imports? I do actually find it helpful for debugging or wanting to look at code to be able to see where things are imported from, so despite this being a bit long, it could prove to be helpful... thoughts?